openra/classes/release.py

```diff
@@ -0,0 +1,21 @@
+
+class Release:
+
+    mod: str
+    version: str
+    is_playtest: bool
+
+    def __init__(self, mod: str, version: str, is_playtest: bool):
+        self.mod = mod
+        self.version = version
+        self.is_playtest = is_playtest
+
+    def __repr__(self):
+        return f'Release({self.__str__()})'
+
+    def __str__(self):
+        is_playtest_text = "Playtest" if self.is_playtest else "Full"
```

Comment:
This works on the assumption that we never have anything like https://github.com/OpenRA/OpenRA/releases/tag/devtest-20210313 . Do we exclude every tag that does not start with `playtest-` or `release-`?

Reply:
It did use regex to figure this out in a previous commit but that is now gone.

The is_playtest is determined by whether it's flagged as a `prerelease` on github. So if a devtest-######## release is made, it'll likely be tagged as that, and will be treated like the playtest versions.

Having said that I don't recall seeing any of these devtest versions getting returned, so I think it'll just be the releases on this page: https://github.com/OpenRA/OpenRA/releases

This is what GitHub says:

https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#list-releases

Reply:
Ok.